content/other/phase-angles/inverse/meta.json

```diff
@@ -0,0 +1,27 @@
+{
+  "abstract": null,
+  "authors": [
+    {
+      "name": "Guillermo Alonso"
```

Comment:
Could we use Guillermo's PennyLane profile here instead?
```suggestion
      "username": "KetPuntoG"
```

Reply:
We should be able to use both. Will update.